### DISCUSSION TITLE
Support for Source Code Management systems?

### DISCUSSION THREAD

**Discussion by JDRay42** [kuafuai/DevOpsGPT#27] [Ideas]
Does DevOpsGPT check-in the code it develops before testing it?  How about after modification due to bugs found during regression testing?

**Comment by booboosui** [comment]
We are developing support for continuous integration systems, and the first phase plans to support pipelining tools for gitlab and GitHub

**Comment by booboosui** [comment]
@JDRay42 we already support Git and look forward to your feedback 
 https://github.com/kuafuai/DevOpsGPT/blob/master/docs/DOCUMENT.md#git-configuration
